src/instructions/tables.rs

```diff
@@ -70,11 +70,18 @@ pub fn sidt() -> DescriptorTablePointer {
 
 /// Load the task state register using the `ltr` instruction.
 ///
+/// Note that loading a TSS segment selector marks the corresponding TSS
+/// Descriptor in the GDT as "busy", preventing it from being loaded again
+/// (either on this CPU or another CPU). TSS structures (including Descriptors
+/// and Selectors) should generally be per-CPU. See
+/// [`tss_segment`](crate::structures::gdt::Descriptor::tss_segment)
+/// for more information.
+///
 /// ## Safety
 ///
 /// This function is unsafe because the caller must ensure that the given
-/// `SegmentSelector` points to a valid TSS entry in the GDT and that loading
-/// this TSS is safe.
+/// `SegmentSelector` points to a valid TSS entry in the GDT, that is entry is
+/// not busy, and that the corresponding data in the TSS is valid.
```

Comment:
That's not a safety concern though, is it? If the entry is busy the CPU fires an exception which is not something we usually consider "unsafe behavior". Perhaps this comment should be moved up.

Reply:
Good catch, raising a `#GP` exception isn't unsafe (it's basically similar to a panic). I moved that comment up.